packages/CalendarCarousel/index.tsx

```diff
@@ -202,6 +226,41 @@ function CalendarCarousel<T>({
       numPrevMonthDays--;
     }
 
+    const markedDays = [7, 17, 27];
+    const markedDayEvents = [
+      {
+        selectedEventDate: new Date(2020, 7, 7),
+        year: 2020,
+        month: 6,
+        day: 7,
+        events: 'walk dog',
+      },
+      {
+        selectedEventDate: new Date(2020, 7, 17),
+        year: 2020,
+        month: 6,
+        day: 17,
+        events: 'birthday',
+      },
+      {
+        selectedEventDate: new Date(2020, 7, 27),
+        year: 2020,
+        month: 6,
+        day: 27,
+        events: 'cooking',
+      },
+    ];
+
+    const [eventSwitch, setEventSwitch] = useState(0);
+    const dayEvents = [];
+    const isInside = (d: number) : boolean => {
+      for (let m = 0; m <= markedDayEvents.length; m++) {
+        if (d === markedDays[m] && month === new Date().getMonth()) {
+          return true;
+        }
```

Comment:
I think using [some in es6 array helper](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/Array/some) would be much better.

You may [read this](https://medium.com/@voonminghann/array-helper-methods-in-es6-412dbc3f845b) at glance!

Reply:
used map and foreach will upload soon!